api/src/opentrons/hardware_control/emulation/util.py

```diff
@@ -31,7 +31,7 @@ def val(self, value: Optional[ValueType]) -> None:
     def __repr__(self) -> str:
         return "none" if self._value is None else str(self._value)
 
-    def __eq__(self, other) -> bool:
+    def __eq__(self, other: Any) -> bool:
```

Comment:
_Might_ make sense to use `object` instead of `Any` here, for a little additional safety

```suggestion
    def __eq__(self, other: object) -> bool:
```